packages/components/paginator/src/paginator-status.ts

```diff
@@ -0,0 +1,113 @@
+import { localized, msg, str } from '@lit/localize';
+import { type DataSource } from '@sl-design-system/data-source';
+import { type CSSResultGroup, LitElement, type PropertyValues, type TemplateResult, html } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import styles from './paginator-status.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-paginator-status': PaginatorStatus;
+  }
+}
+
+/**
+ * A component that can be used with the paginator component.
+ * Contains information about currently visible items on the page and total amount of items.
+ */
+@localized()
+export class PaginatorStatus extends LitElement {
+  /** @internal */
+  static override styles: CSSResultGroup = styles;
+
+  /** @internal Pages amount. */
+  #pages = 1;
+
+  /** Currently active page, if not set - default to 1. */
+  @property({ type: Number, attribute: 'active-page' }) activePage = 1;
+
+  /** @internal Currently visible items on the current page. */
+  @state() currentlyVisibleItems = 1;
+
+  /** Provided data source. */
+  @property({ attribute: false }) dataSource?: DataSource;
+
+  /** Items per page, if not set - default to 10. */
+  @property({ type: Number, attribute: 'items-per-page' }) itemsPerPage = 10;
+
+  /** Total amount of items, if not set - default to 1. */
+  @property({ type: Number }) total = 1;
```

Comment:
`pageCount`

Reply:
So i read the comment wrong: `totalItems` better reflects the property i think. And also makes clear it's unrelated to pageCount